code/WorkInProgress/actuallyKeelinsStuff.dm

```diff
@@ -94,6 +94,20 @@ Returns:
 			I.filters += filter(type="layer", render_source = (islist(render_source_cap) ? pick(render_source_cap) : render_source_cap), transform=M2)
 		I.transform = UNLINT(matrix().Turn(-angle).Translate((dist),0).Turn(angle))
 		result.lineImage = I
+	else if(mode == LINEMODE_STRETCH_NO_CLIP)
```

Comment:
document why this exists and is different to `LINEMODE_STRETCH` (since the name implies it should just be the default behaviour of `LINEMODE_STRETCH`)

Reply:
fair, see if the new docs are satisfactory!